etc/helm/test/local_test.go

```diff
@@ -71,6 +73,17 @@ func TestLocal(t *testing.T) {
 				continue
 			}
 			checks["headless service"] = true
+
+		case *v1.Secret:
+			if object.Name == "pachyderm-config" {
```

Comment:
We might want to consider renaming the secret to `pachyderm-auth-config` to make way for a global `pachyderm-config` secret object

Reply:
this is an interesting one. Although most of the config-pod data is auth related (IDP connectors, etc...), it doesn't have to be strictly so. So I am wondering whether a future `pachyderm-config` may still be related to the config-pod. In the case that it's not, maybe we could call this one `pachyderm-config-job-config` ?

Reply:
based on conversation in slack, we'll go with `pachyderm-bootstrap-config`